### PR TITLE
Adds ML-41 ICC Autoshotgun

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds ML-41 Autoshotgun to ICC guards.
Adds 12G Frag rounds, fires 4 pellets that do 2 weak explosion on hit.
image

Gun stats below
<details>

Fire delay-> .9
Damage multiplier-> .5

Aim slowdown-> 0.6
Magazine Size-> 16
Wield delay-> 0.55

Only has flechette and frag rounds.
</details>

## Why It's Good For The Game
More interesting cool stuff for ERTs, also https://www.youtube.com/watch?v=_0-QRmtlLAQ

## Changelog
:cl:
add: Added new 12G Frag rounds, only for ICC so far.
add: New ML-41 12G autoshotgun, it can fire 12G flechettes and fragmentation rounds.
/:cl:

### CODE REVIEW

**Comment by Wirpy** [comment]
lets go

**Comment by Mrrpip** [comment]
Okie-dokie. Pondering, as they're 12G and as such would fit in other shotguns.

Would it be possible to have a placeholder for the Explosive shells as loose ammo?

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -2396,3 +2396,56 @@
 
 /obj/item/weapon/gun/rifle/icc_confrontationrifle/leader
 	starting_attachment_types = list(/obj/item/attachable/lasersight, /obj/item/attachable/magnetic_harness, /obj/item/attachable/extended_barrel)
+
+//-------------------------------------------------------
+//ML-41 Autoshotgun
+
+/obj/item/weapon/gun/rifle/icc_autoshotgun
+	name = "\improper ML-41 autoshotgun"
+	desc = "The ML-41 Automatic Shotgun is used by the ICCAF in fast paced boarding assaults, fielding a wide variety of ammo for all situations. Takes 20-round 12 gauge drums."
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/rifles.dm:2405]:
```suggestion
	desc = "The ML-41 Automatic Shotgun is used by the ICCAF in fast paced boarding assaults, fielding a wide variety of ammo for all situations. Takes 16-round 12 gauge drums."
```
20 or 16 m8?

**Review by Lumipharon — APPROVED** [review]
This sounds horribly strong in certain situations but uh, ert weapon I guess. It being 12ga means marines should be able to load them if they get them however, which is funny.

**Pull request merged**